packages/cozy-harvest-lib/src/components/TriggerManager.jsx

```diff
@@ -218,6 +199,32 @@ export class DumbTriggerManager extends Component {
     return account
   }
 
+  async componentDidMount() {
+    const {
+      konnector,
+      showUnlockForm,
+      onVaultDismiss,
+      vaultClosable,
+      vaultClient
+    } = this.props
+    const konnectorPolicy = findKonnectorPolicy(konnector)
+    const isVaultLocked = await vaultClient.isLocked()
+    if (konnectorPolicy.saveInVault) {
+      window.vaultClient = vaultClient
```

Comment:
Is it your new way to debug your code ? :D

Reply:
Yes ;) Removed, thanks